packages/extensions/openapi-to-typespec/src/utils/model-generation.ts

```diff
@@ -72,3 +82,64 @@ export function generateTemplateModel(templateModel: TypespecTemplateModel): str
 export function generateAdditionalProperties(properties: TypespecObjectProperty[]): string {
   return properties.map((p) => getModelPropertyDeclarations(p).join("\n")).join(";");
 }
+
+const commonTypeMapping: Record<string, string> = {
+  Resource: "Azure.ResourceManager.CommonTypes.Resource",
```

Comment:
consider to remove the namespace if they are always the same

Reply:
I want to keep this in case they have different namespace.